### PR TITLE
Fixes the failed test due to replacement of bootstrap-tags input with select2

### PR DESCRIPTION
**Pull request by Prerna-0202**
Fixes #

#### Describe the changes you have made in this PR -
I have fixed the test failed due to the pull request #3364 

### Screenshots of the changes (If any) -

Screenshot (863)

Note: Please check **Allow edits from maintainers.** if you would like us to assist in the PR.

### CODE REVIEW

File: app/helpers/utils.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def self.parse_mails_except_current_user(mails, current)
   # @param newly_added array of emails that have been newly added
   # @return notice string
   def self.mail_notice(input_mails, parsed_mails, newly_added)
-    total = input_mails.count(&:present?)
+    total = input_mails.select(&:present?).count
```
[the remark this answers is not in the document]
Comment (tachyons) [reply app/helpers/utils.rb:25]: Could you revert this change as specs now passing params in array format?

[the remark this answers is not in the document]
Comment (Prerna-0202) [reply app/helpers/utils.rb]: yes done

**Review by tachyons — CHANGES_REQUESTED** [review]

File: app/views/groups/_add_member_modal.html.erb [hunk 2]
```diff
@@ -25,7 +25,7 @@
           </div>
           <div class="form-group">
             <%= form.label :emails %>
-            <%= form.select :emails,[ ],{},{ class: 'form-select', multiple: true } %>
+            <%= form.select :emails,["example@gmail.com"],{},{ class: 'form-select', multiple: true } %>
```
[the remark this answers is not in the document]
Comment (Prerna-0202) [reply app/views/groups/_add_member_modal.html.erb]: Done @tachyons

File: spec/system/group_spec.rb [hunk 3]
```diff
@@ -36,8 +36,11 @@
     click_button "+ Add Members"
     execute_script "document.getElementById('addmemberModal').style.display='block'"
     execute_script "document.getElementById('addmemberModal').style.opacity=1"
-    fill_in "group_email_input", with: @user2.email
-    fill_in "group_email_input", with: " "
+    execute_script "var new_email = document.createElement('option')
```
Comment (tachyons) [inline spec/system/group_spec.rb:39]: This is a hack, we should be able to do this without this hack, I am merging this now to unlblock CI. Could you create a followup issue to simplify this ?

**Review by tachyons — APPROVED** [review]

File: spec/system/group_spec.rb (hunk 3 above)
Comment (Prerna-0202) [reply spec/system/group_spec.rb:39]: yaa sure

**Pull request merged**